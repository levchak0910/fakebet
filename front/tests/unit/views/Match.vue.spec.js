import {createWrapper, disableFile} from "../__utils__";
import Component from "@/views/Match.vue";

disableFile();

describe("Match.vue", () => {

    let wrapper;
    let mountWrapper = (options = {}) => {
        wrapper = createWrapper(Component, {
            stubs: ["app-match-menu", "app-match-actions"],
            ...options,
        });
    };

    describe("Testing snapshot", () => {

        it("Component itself", () => {
            mountWrapper();

            expect(wrapper.element).toMatchSnapshot();
        });

    });

});
