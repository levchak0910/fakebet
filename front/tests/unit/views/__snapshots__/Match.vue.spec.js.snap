// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match.vue Testing snapshot Component itself 1`] = `
<div>
  <app-match-menu-stub />
  <app-match-actions-stub />
</div>
`;
